t/ui/15-comments.t

```diff
@@ -293,6 +293,17 @@ subtest 'editing when logged in as regular user' => sub {
     };
 };
 
+subtest 'add comment pinned as group description' => sub {
+    $driver->get($baseurl . 'group_overview/1001');
+    my @comments = $driver->find_elements('#group_descriptions.comment-row', 'css');
+    is(scalar @comments, 0, 'there shouldn\'t appear any descriptions so far');
```

Comment:
just keep it simple and write 'there should not appear any …' so you don't need to escape the apostrophe